### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by meichstedt**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,32 +1,20 @@
 ---
 name: Bug report
-about: Report a bug encountered while operating KUDO
+about: Report a bug on the KUDO docs site
 title: ''
-labels: kind/bug
-assignees: ''
+labels: ''
+assignees: meichstedt
```
Comment (alenkacz) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: 🎉 welcome to the team. Are you going to be joining all the meetings? :D

File: .github/ISSUE_TEMPLATE/enhancement-request.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 name: Enhancement Request
-about: Suggest an enhancement to the KUDO project
+about: Suggest an enhancement to the KUDO site
 title: ''
-labels: kind/enhancement
+labels: ''
 assignees: ''
```
Comment (alenkacz) [inline .github/ISSUE_TEMPLATE/enhancement-request.md]: not Matthias? :D

**Review by alenkacz — APPROVED** [review]
🚢

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (meichstedt) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Yes!

File: .github/ISSUE_TEMPLATE/enhancement-request.md (hunk 2 above)
Comment (meichstedt) [reply .github/ISSUE_TEMPLATE/enhancement-request.md]: Ok ok :)

**Pull request merged**